boards/acd52832/include/periph_conf.h

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2016-2017 Freie Universität Berlin
+ *                    2017 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_acd52832
+ * @{
+ *
+ * @file
+ * @brief       Peripheral configuration for the ACD52832
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Dimitri Nahm <dimitri.nahm@haw-hamburg.de>
+ *
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Clock configuration
+ *
+ * @note    The radio will not work with the internal RC oscillator!
```

Comment:
Hmmm... this sounds familiar from the EFM family CPUs, can you take a look on how it was solved?

Reply:
Will check. Can you reference a discussion?

Reply:
Yes, you can find @basilfx PR [here](https://github.com/RIOT-OS/RIOT/pull/8165/files).

Reply:
I don't see how this is related. I just checked the reference manual which states `The HFXO must be running to use the RADIO` -> so this is a hardware 'limitation' and nothing to address here.

Reply:
I meant that the clock configuration for the whole family can be factorised and generalised, but well... it can come in a following up PR...

Reply:
Ahhh ok. Well that is an other feature which I don't want to address here. But yes, good idea in general

Reply:
>  meant that the clock configuration for the whole family can be factorised and generalised, 

This is already done: the two values here are actually needed to be defined by each board, as only the boards knows about its used oscillators...